srtcore/congctl.h

```diff
@@ -156,19 +156,17 @@ class SrtCongestionControlBase
     // If not, it will be internally calculated.
     virtual int RTO() { return 0; }
 
-    // How many packets to send one ACK, in packets.
-    // If user-defined, will return nonzero value.  It can enforce extra ACKs
-    // beside those calculated by ACK, sent only when the number of packets
-    // received since the last EXP that fired "fat" ACK does not exceed this
-    // value.
-    virtual int ACKInterval() { return 0; }
-
-    // Periodical timer to send an ACK, in microseconds.
-    // If user-defined, this value in microseconds will be used to calculate
+    // ACK interval in packets.
+    // Specifies the number of packets to receive before sending the ACK.
+    // Used by CUDT together with ACKIntervalMicrosec() to trigger ACK packet sending.
+    virtual int ACKIntervalPkts() const { return 0; }
+
+    // Periodical interval to send an ACK, in microseconds.
+    // If user-defined, this value will be used to calculate
     // the next ACK time every time ACK is considered to be sent (see CUDT::checkTimers).
     // Otherwise this will be calculated internally in CUDT, normally taken
-    // from CPacket::SYN_INTERVAL.
-    virtual int ACKPeriod() { return 0; }
+    // from CUDT::m_ullACKInt_tk.
```

Comment:
`CPacket` structure does not have `SYN_INTERVAL`. And `CUDT::m_ullACKInt_tk` is what is actually used, even though it is initialized with `CUDT::COMM_SYN_INTERVAL_US`.

Reply:
The comment uses the word "normally" (meaning, by default, in case of no special settings) and its intention was to refer to the exact point where the value is set, not one of the intermediate fields. Besides, I wrote this! :D

Reply:
Replaced with `CUDT::COMM_SYN_INTERVAL_US`